File: Lista1/src/exec3.java
/**
 *
 * @author 11800910
 */
public class exec3 {

    public static void main(String[] args) {
        System.out.println("EXERCICIO 3: Arvore esquerda->direita\n");
        int num = 0;
        boolean leu = false;
        while(!leu){
            leu = true;
            System.out.printf("Digite um inteiro: ");
            try {
                num = EntradaTeclado.leInt();
                if( num < 0){
                    System.out.println("Número deve ser positivo!!");
                    leu = false;
                }
            } catch (Exception e) {
            System.out.println("O valor que vc digitou não é um número inteiro");
                leu = false;
            }
        }
        // com num validado, vamos montar a arvore
        int x = num;
        while(x > 0){
            for(int i = 0; i < x; i++)
                System.out.printf("*");
            System.out.printf("\n");
            x--;
        }
    }
}

File: Lista2/exec3.py
'''
 *
 * @author 11800910
'''

def valida_entrada():
    while(True):
        try:
            num = int(input('Digite um inteiro: '))
            break
        except ValueError:
            print('O valor que vc digitou não é um número inteiro')
    return num

if __name__ == '__main__':

    # cabeçalho
    print('EXERCICIO 3:Arvore esquerda->direita\n')
    
    # valdação da entrada 
    num = valida_entrada()

    # com num validado, vamos montar a arvore
    x = num
    # loop vertical (até acabar o num)
    while x > 0:
        # loop horizontal (até acabar a linha)
        for _ in range(0, x):
            print('*', end="")
        print('\n')
        x -= 1
